File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int n = nums.size();
        bool flag = false;
        sort(nums.begin(), nums.end());

        for(int i = 1; i < n; i++) {
            if(nums[i-1] == nums[i])
                flag = true;
        }
        return flag;
    }
};

File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] arr = Arrays.copyOf(nums,nums.length*2);
        for(int i = 0; i < nums.length; i++) {
            arr[i + nums.length] = nums[i];
        }
        return arr;
    }
}